src/components/AttentionBox/__stories__/attentionBox.stories.mdx

```diff
@@ -0,0 +1,162 @@
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import AttentionBox from "../AttentionBox";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { ATTENTION_BOX_TYPES } from "../AttentionBoxConstants";
+import { MultipleStoryElementsWrapper, Link } from "../../../storybook/components";
+import DialogContentContainer from "../../DialogContentContainer/DialogContentContainer";
+import { Info, Invite } from "../../Icon/Icons";
+import Button from "../../Button/Button.jsx";
+import Icon from "../../Icon/Icon";
+import Search  from "../../Search/Search";
+import Avatar from "../../Avatar/Avatar";
+import { TOOLTIP, ALERT_BANNER, TOAST } from "../../../storybook/components/related-components/component-description-map";
+import person from "./assets/person.png"
+import "./attentionBox.stories.scss";
+
+<Meta
+  title="Components/AttentionBox"
+  component={ AttentionBox }
+/>
+
+<!--- Component template -->
+
+export const attentionBoxTemplate = createComponentTemplate(AttentionBox);
+
+<!--- Component documentation -->
+
+# Attention Box
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Attention box lets users know important information within content areas, as close as possible to the content it’s about.
+<Canvas>
+  <Story name="Overview"
+         args={{ title: "Attention box title", text: "Studies show that 100% of people who celebrate birthdays, will eventually die.", componentClassName: "monday-style-attention-box_box" }}>
+    { attentionBoxTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ AttentionBox } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Use attention box if there is crucial information for user to finish or be acknowledged about a task.",
+  "Attention box do not dismiss automatically. They persist on the page until the user dismisses them or takes action that resolves the notification.",
+  "The width of attention box is based on content and layout. They can expand to the fill the container or content area they relate to.",
+  "Their height is based on the content length, which should not exceed two lines of text.",
+]}/>
+
+<Tip title="Check yourself">If the information about a component isn’t critical for the user, use a <Link href="/?path=/docs/components-tooltip--overview" size={Link.sizes.SMALL} withoutSpacing>Tooltip</Link> instead.</Tip>
+
+## Variants
+### States
+There are 4 types of attention boxes: Primary, success, danger, and dark (natural).
```

Comment:
please change 4 to "four" (and also any numeric representation of a number in text which smaller than 5 to text representation because this is our one of our copywriting standards in monday)